src/domUpdates.js

```diff
@@ -290,33 +301,50 @@ const createIngredientsHTML = ingredients => {
 
 const switchView = (clickedViewID) => {
   if (clickedViewID === "our-recipes") {
-    recipeGrid.classList.remove("hidden");
-    ourViewBtn.classList.add("selected-view");
-    yourViewBtn.classList.remove("selected-view");
-    yourViewBtn.classList.add("unselected-view");  
-    ourViewBtn.classList.remove("unselected-view");
+    pageData.recipesOfInterest = copyItem(pageData.allRecipes);
   } else {
-    recipeGrid.classList.add("hidden");
-    ourViewBtn.classList.remove("selected-view");
-    yourViewBtn.classList.add("selected-view");
-    yourViewBtn.classList.remove("unselected-view");  
-    ourViewBtn.classList.add("unselected-view");
+    pageData.recipesOfInterest = copyItem(currentUser.recipesToCook);
   }
+  pageData.currentView = clickedViewID
+  renderGrid(pageData.recipesOfInterest)
+  toggleView([ourViewBtn, yourViewBtn])
 }
 
 const searchForRecipes = () => {
-  const data = {
-    ourRecipes: pageData.recipesOfInterest,
-    yourRecipes: currentUser.recipesToCook
-  }
-  let searchedRecipes = searchRecipes(data[pageData.currentView], ingredientsData, searchBar.value)
+  let searchedRecipes = searchRecipes(pageData.recipesOfInterest, ingredientsData, searchBar.value)
   if(searchedRecipes) {
-    searchedRecipes.length  
-    ? renderGrid(searchedRecipes)
-    : recipeGrid.innerHTML = `<p>Sorry, we couldn't find any recipes for your search of "${searchBar.value}"</p>`
+    if(searchedRecipes.length) {
+      pageData.recipesOfInterest = searchedRecipes;
+      renderGrid(searchedRecipes)
+    }else {
+      recipeGrid.innerHTML = `<p>Sorry, we couldn't find any recipes for your search of "${searchBar.value}"</p>`
+    }
+  }
+}
+
+const updateUserRecipes = (e) => {
+  if(e.target.parentNode.classList.contains('panel')) {
```

Comment:
Done - 
- added a class of `save-option` to acceptable targets instead of using closest in order to avoid having this function fire off unintendedly, and instead of parentNode to allow for future changes 
- changed inner if statements to add or delete based or recipe save status (move towards working off DM instead of DOM)